satellite-clone-vars.sample.yml

```diff
@@ -51,11 +33,24 @@ org: changeme
 # uncomment the following line to override this option (defaults to true)
 #run_pre_install_check: true
 
-# If register_to_portal is set to false:
+# Use register_to_portal to automatically register to Red Hat Portal. By default,
 #   - the host will not be registered to RH portal.
-#   - it is the responsibility of the user to make the required repos available.  Otherwise, the script will stop with an error.
-# uncomment the following line to override this option (defaults to true)
-#register_to_portal: true
+#   - it is the responsibility of the user to make the required repos available.
+# uncomment the following line to override this option (defaults to false). If you override this option, you must update
+# `activationkey` and `org` variables in the next section.
+#register_to_portal: false
+
+# Red Hat Portal Registration info - Credentials to register your Satellite to Red Hat portal
```

Comment:
I think it would be a good idea to reinforce that you only need this section if "register_to_portal" is true here

Reply:
sure, added ```Use this section if and only if `register_to_portal` is `true`.```